bots/image-refresh

```diff
@@ -18,19 +18,23 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+import logging
 import os
 import subprocess
 import sys
+import time
 
 import task
 from task import github, REDHAT_STORE
 
 TRIGGERS = {
     "centos-7": [
-        "verify/centos-7"
+        "verify/centos-7",
+        "cockpit/centos-7@cockpit-project/starter-kit",
```

Comment:
The commit message should describe this change, i. e. that external tests now need to be explicitly enumerated in image-refresh.